pkgs/applications/graphics/inkscape/default.nix

```
@@ -182,14 +168,34 @@ stdenv.mkDerivation rec {
     done
   '';
 
-  passthru.tests.ps2pdf-plugin = callPackage ./test-ps2pdf-plugin.nix { };
+  passthru = {
+    tests = {
+      ps2pdf-plugin = callPackage ./test-ps2pdf-plugin.nix { };
+      inkex = python3Packages.inkex;
```

We already have `python3` in scope so it will be cleaner to do

```suggestion
      inherit (python3.pkgs) inkex;
```

The difference from `python3Packages` does not matter in this context.

---

Ok.